test/test-utils.js

```diff
@@ -25,7 +25,20 @@ const stdout = () => {
 
 };
 
+const isExtractPresent = (Metrics) => {
+	const metrics = Object.keys(Metrics);
+	let isExtract = true;
+	metrics.forEach((metric) => {
+		const { extract } = Metrics[metric];
+		if (extract === undefined) {
+			isExtract = false;
+		}
+	});
+	return isExtract;
```

Comment:
```suggestion
    return metrics.every((metric) => typeof Metrics[metric].extract !== 'undefined')
```

altho i'm not sure if we need this as a utility, rather than just inline in the test?

Reply:
Got it! And yeah you this can be an inline test now